src/main/java/compsci4039/week4/exceptions/Lab8SavingsAccount.java

```diff
@@ -0,0 +1,23 @@
+package compsci4039.week4.exceptions;
+
+public class Lab8SavingsAccount extends Lab8BankAccount {
+
+    private double interestRate;
+
+    public Lab8SavingsAccount(Lab8Person person, double interestRate) {
+        super(person);
+        this.interestRate = interestRate;
+    }
+
+    @Override
```

Comment:
Just a note:
I don't know what that is and how it works :)

Reply:
@Arden488 

The ```@Override``` notation makes it possible to re-implement some public/protected method already declared in a parent Class. We see this a lot when we add functionalities to the ```toString()``` or ```equals``` methods. 

You can completely change how it works (bypassing the original inherited behavior) or you can add something on the top of what has been inherited -- calling ```super()```. 😄